src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="container h-75"
+     *ngIf="!startClusterCreation">
+  <div class="row h-100 justify-content-center align-items-center">
+    <div class="blank-page">
+      <!-- htmllint img-req-src="false" -->
+      <img [src]="projectConstants.cephLogo"
+           alt="Ceph"
+           class="img-fluid mx-auto d-block">
+      <h3 class="text-center m-2"
+          i18n>Welcome to {{ projectConstants.projectName }}</h3>
+
+      <div class="m-4">
+        <h4 class="text-center"
+            i18n>Please expand your cluster first</h4>
+        <div class="offset-md-2">
+          <button class="btn btn-accent m-2"
+                  name="expand-cluster"
+                  (click)="createCluster()"
+                  i18n>Expand Cluster</button>
```

Comment:
```suggestion
                  i18n>Start</button>
```
or
```suggestion
                  i18n>Continue</button>
```

Reply:
I was following the design agreement we made on https://marvelapp.com/prototype/5ed82bc/screen/73191921. IMHO, it makes more sense to keep this button named as Expand Cluster rather than `Continue` or `Start`.